docker/docker-compose-btcd.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+services:
+    # btc is an image of bitcoin node which used as base image for btcd and
+    # btccli. The environment variables default values determined on stage of
+    # container start within starting script.
+    btcd:
+      image: btcd
+      container_name: btcd
```

Comment:
And now we have two files for `btcd`???

Reply:
> I think we should keep the `docker-compose.yml` with `btcd` to not break users that are already using it. So just add a new `docker-compose-bitcoind.yml` instead.

Okay I somehow understood your review incorrectly. I will delete this new btcd file and remain the original.

Reply:
Sorry, It seems I've mistakenly replaced the `docker-compose-bitcoind` with `docker-compose-btcd`! Returned it...